pages/manual/planet/overview.md

```diff
@@ -52,26 +52,30 @@ When you are inside PLANET you will see different features available.
 List of features:
 1. Library
 2. Courses
-3. Surveys
-4. Teams
-5. Meetups
-6. News
-7. Achievement
-8. Feedback
-9. Manager
+3. Exams
+4. Surveys
+5. Teams
+6. Meetups
+7. News
+8. Achievement
+9. Feedback
+10. Manager
 
 ## Library
 The Library is home to the community's collection of media and other resources. Click [here](library.md) to learn more about Library.
 
 ## Courses
 The Courses page is where you can create and join a variety of different courses where you can learn many different topics. Click [here](./course.md) to learn more about Courses.
 
-## Teams
-Here you will be able to create teams to communicate with them, offer courses, and organize members. Click [here](teams.md) to learn more about Teams
+## Exams
```

Comment:
Oops let not say click here as it will not take anywhere any people will just raise issue.
Instead Just say that this portion needs to be worked on.